python_modules/dagster/dagster_tests/definitions_tests/auto_materialize_tests/test_perf.py

```diff
@@ -0,0 +1,41 @@
+import time
+
+import pytest
+from dagster import (
+    DailyPartitionsDefinition,
+    Definitions,
+    HourlyPartitionsDefinition,
+    SchedulingCondition,
+)
+from dagster_test.toys.auto_materializing.large_graph import AssetLayerConfig, build_assets
+
+from dagster_tests.conftest import IS_BUILDKITE
+from dagster_tests.definitions_tests.auto_materialize_tests.test_user_space_ds_api import (
+    execute_ds_ticks,
+)
+
+
+@pytest.mark.skipif(IS_BUILDKITE, reason="Temporary")
```

Comment:
this gets dropped up stack i assume ?

Reply:
yep! specifically in 59/n (top of the stack)